var/spack/repos/builtin/packages/draco/package.py

```diff
@@ -63,11 +72,3 @@ def cmake_args(self):
             '-DBUILD_TESTING={0}'.format('ON' if self.run_tests else 'OFF')
         ])
         return options
-
-    @run_after('build')
-    @on_package_attributes(run_tests=True)
-    def check(self):
-        """Run ctest after building project."""
-
-        with working_dir(self.build_directory):
-            ctest()
```

Comment:
Why was the test removed? Is `make test` equivalent?

Reply:
Correct.  It was equivalent to `make test`.